src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days < 3) {
+    return totalCost;
+  } else if (days >= 3 && days <= 6) {
```

Comment:
Prefer `if` with `return` over `if else` to simplify the code. Since you are returning inside the `if`, there is no need for the `else` keyword.